src/Layout/Footer.tsx

```diff
@@ -21,11 +21,17 @@ export interface FooterProps {
  */
 const Footer = styled.footer<FooterProps>`
   align-items: baseline;
-  background-color: ${({ background }) => background};
+  background: ${({ theme }) => theme.color.background.light};
   display: flex;
   justify-content: ${({ justify }) => justify};
   padding: ${({ theme }) => `${theme.ws.medium} ${theme.ws.small}`};
   width: 100%;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 40px;
+  font-size: 0.9em;
+  font-weight: 400;
```

Comment:
Rather than hard code this, I'd suggest using the `theme` values here instead. Feel free to tweak these as needed to whatever you feel is more appropriate (i.e: `theme.font.bold.size` etc. - vs code should be able to typehint the available options for you)

```suggestion
  font-size: ${({ theme }) => `${theme.font.base.size}`};
  font-weight: ${({ theme }) => `${theme.font.base.weight}`};
```